azure-pipelines.yml

```diff
@@ -114,7 +114,12 @@ jobs:
     - bash: |
         set -e
         sudo apt-get update
-        sudo apt-get -y -q --no-install-recommends install zlib1g-dev libncursesw5-dev libgeoip-dev nettle-dev libgmp-dev libcurl4-gnutls-dev libsdl2-dev libogg-dev libvorbis-dev libopusfile-dev libwebp-dev libjpeg8-dev libpng-dev libfreetype6-dev libglew-dev libopenal-dev liblua5.2-dev ninja-build $(EXTRA_PACKAGES)
+        sudo apt-get -q install -y gdebi-core
+        wget http://mirrors.kernel.org/ubuntu/pool/universe/g/glew/libglew2.2_2.2.0-4_amd64.deb
+        wget http://mirrors.kernel.org/ubuntu/pool/universe/g/glew/libglew-dev_2.2.0-4_amd64.deb
+        sudo gdebi -q -n libglew2.2_2.2.0-4_amd64.deb
+        sudo gdebi -q -n libglew-dev_2.2.0-4_amd64.deb
```

Comment:
Leave comments about why we need this and when we can remove it.

Reply:
Done in another pr, this one's gonna be rebased on master instead.